src/chess/Command.java

```diff
@@ -18,12 +18,13 @@
   static Command parse(String str) {
     switch (str.toLowerCase()) {
       case "help":
-        System.out.println("* type 'help' for help");
-        System.out.println("* type 'board' to see the board again");
-        System.out.println("* type 'resign' to resign");
-        System.out.println("* type 'moves' to list all possible moves");
-        System.out.println("* type a square (e.g. b1, e2) to list possible moves for that square");
-        System.out.println("* type a UCI (e.g. b1c3, e7e8q) to make a move");
+        System.out.println("* type 'help' for help \n " +
+            "* type 'board' to see the board again \n " +
+            "* type 'resign' to resign \n " +
+            "* type 'moves' to list all possible moves \n " +
+            "* type a square (e.g. b1, e2) to list possible moves for that square \n " +
+            "* type a UCI (e.g. b1c3, e7e8q) to make a move");
+
```

Comment:
This modification itself is really nice!

But I noticed that I missed something I had to mention in your previous PR.

You put this printing in `parse()` method of Command enum, but as the name of this method shows, you should only do things related to parsing a string to Command. So you might find that printing out something is not related to this method.

You should re-think where to put these code. Let's try💪

Reply:
This is not just a recommendation, but a request!